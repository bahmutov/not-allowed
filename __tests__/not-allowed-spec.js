'use strict'

/* eslint-env jest */
const notAllowed = require('..')

describe('not-allowed', () => {
  it('is a function', () => {
    expect(typeof notAllowed).toBe('function')
  })

  it('throws error', () => {
    expect(notAllowed).toThrowErrorMatchingSnapshot()
  })

  it('throws error with arguments', () => {
    const fn = () => notAllowed('foo', 'bar', 42)
    expect(fn).toThrowErrorMatchingSnapshot()
  })
})
