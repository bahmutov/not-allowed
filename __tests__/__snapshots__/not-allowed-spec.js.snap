// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`not-allowed throws error 1`] = `
"Not allowed to call this function with arguments
    
  "
`;

exports[`not-allowed throws error with arguments 1`] = `
"Not allowed to call this function with arguments
    foo bar 42
  "
`;
